apps/crn-server/src/validation/research-output.validation.ts

```diff
@@ -79,6 +79,7 @@ const researchOutputPostRequestValidationSchema: JSONSchemaType<ResearchOutputPo
       sharingStatus: { type: 'string' },
       usedInPublication: { type: 'boolean', nullable: true },
       publishDate: { type: 'string', format: 'date-time', nullable: true },
+      publishingEntity: { type: 'string', nullable: true },
```

Comment:
use an enum, similarly to https://github.com/yldio/asap-hub/blob/cf8b9a2bb3df706e08bab045f6cbf2ef95387094/apps/crn-server/src/validation/research-output.validation.ts#L45